pkg/cmd/taskrun/list.go

```diff
@@ -108,6 +109,9 @@ List all TaskRuns of Task 'foo' in namespace 'bar':
 			}
 
 			if trs != nil {
+				if opts.Reverse {
```

Comment:
Same comment here as for pipelineruns:

Checking `opts.Reverse` here misses out on situations when `--output` or `-o` would be used. 

You may want to check for `if opts.Reverse` on [line 87](https://github.com/tektoncd/cli/blob/64f3fa7c03b43e3d75d8de86adacd9728b794259/pkg/cmd/taskrun/list.go#L87). This way `--output` situations would also be included. You would also need a nil check for `trs`:

```go
if trs != nil && opts.Reverse {
    reverse(trs)
}
```

It would also be nice to add a unit test for this case when `-o` is specified with `--reverse`.